homework/index.js

```diff
@@ -28,27 +28,74 @@
     });
     return elem;
   }
+  function NewDate(dateString) {
```

Comment:
Function, variable and parameter names should be in `camelCase`: `newDate`. Moreover, `newDate` is not the best possible name. You are not creating a new date, you are just formatting an existing date for presentation purposes. Therefore, the name `formatDate` would seem a better fit.

Reply:
Firstly Thank you for this feedback. Yes, you right about that. I'm going to change its name